src/pages/trader-tools/common/_validation.js

```diff
@@ -75,10 +75,30 @@ const validation = {
     swapRate: (input) => {
         return numberWithNegativeValidation(input, localize('Swap rate'), 15)
     },
-
     pointValue: (input) => {
         return numberValidation(input, localize('Point value'), 15)
     },
+    takeProfitAmount: (input) => {
+        return numberValidation(input, localize('Take profit amount'), 15)
+    },
+    stopLossAmount: (input) => {
+        return numberValidation(input, localize('Stop loss amount'), 15)
+    },
+    takeProfitLevel: (input) => {
+        return numberValidation(input, localize('Take profit level'), 15)
+    },
+    stopLossLevel: (input) => {
```

Comment:
I think stoploss and takeprofit level needs to be checked,
stoploss won't be higher than take profit level isnt it

Reply:
As discussed, the amounts will probably not have these validations but more likely on the levels instead. Let's include these in the next phase once we get more clear requirements on these.